### PR TITLE
Add manualDependencies feature, and use it for NSS

### PR DESCRIPTION
**Pull request by chaen**
This fixes https://github.com/DIRACGrid/DIRACOS/issues/112
It was tested here: https://gitlab.cern.ch/CLICdp/iLCDirac/diracos-test/pipelines/1354718

BEGINRELEASENOTES
NEW: Add manualDependencies feature and use it for NSS
NEW: postfix package is now ignored
NEW: add an ldd test

ENDRELEASENOTES

### CODE REVIEW

File: tests/integration/test_ldd.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+newMissingDependencies=$(sort $testDir/knownMissingDependencies.txt | comm -13 - $DIRACOS/missingDependencies.txt );
+if [ ! -z "$newMissingDependencies" ];
+then
+  echo "New missing dependencies"
+  echo $newMissingDependencies;
+  rc=1;
+else
+  echo "All OK";
+  echo "Known missing dependencies that are not missing anymore:"
+  sort $testDir/knownMissingDependencies.txt | comm -23 - $DIRACOS/missingDependencies.txt
```
Comment (chrisburr) [inline tests/integration/test_ldd.sh]: Should this be an error to make it easier to notice when an intentionally missing package is included?

Comment (chaen) [reply tests/integration/test_ldd.sh]: hum, I did not think of that use case I must say. Good idea

Comment (chaen) [reply tests/integration/test_ldd.sh]: Done

**Comment by chaen** [comment]
@cburr better ?

**Pull request merged**